File: Bailaconmigo/src/main/java/org/example/bailaconmigo/DTOs/UpcomingEventReportDto.java
package org.example.bailaconmigo.DTOs;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class UpcomingEventReportDto {
    private Long eventId;
    private String eventName;
    private LocalDateTime eventDate;
    private String organizerName;
    private String cityName;

    private Integer capacity;
    private int currentRegistrations;
    private int availableSpots;
    private double occupancyRate;
}

File: frontend/src/app/models/upcoming-event-report-dto.ts
export interface UpcomingEventReportDto {
  eventId: number;
  eventName: string;
  eventDate: string;
  organizerName: string;
  cityName: string;
  capacity: number;
  currentRegistrations: number;
  availableSpots: number;
  occupancyRate: number;
}
